src/animation/PropertyBinding.js

```diff
@@ -496,6 +496,25 @@ class PropertyBinding {
 
 					break;
 
+				case 'map':
+
+					if ( ! targetObject.material ) {
+
+						console.error( 'THREE.PropertyBinding: Can not bind to material as node does not have a material.', this );
```

Comment:
I think "bind to material map", not "bind to material"?

Reply:
The error message should be correct since the check happens in context of `material`.